app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -36,6 +36,10 @@ CONFIG_ZEPHYR_NATIVE_DRIVERS=y
 # SOF / loadable modules
 CONFIG_INTEL_MODULES=y
 CONFIG_LIBRARY_MANAGER=y
+CONFIG_LIBRARY_BASE_ADDRESS=0xa0688000
+CONFIG_LIBRARY_MANAGER=y
+CONFIG_LIBRARY_BUILD_LIB=y
+CONFIG_LIBRARY_DEFAULT_MODULAR=y
```

Comment:
Hmm, is this series missing something still? I don't see ptl build changed with this PR, should it?  I see ace30_ptl.conf doesn't have LLEXT set, so this CONFIG_LIBRARY_DEFAULT_MODULAR=y setting won't have any impact. Is this intentional?

Reply:
@kv2019i no, it isn't intentional, thanks for catching! Let me fix